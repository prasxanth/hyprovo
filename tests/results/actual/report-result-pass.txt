2021-11-24 22:51:24.290 - INFO - Pass âœ“...  Passing result
