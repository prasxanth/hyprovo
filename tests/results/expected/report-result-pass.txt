2021-11-24 22:50:31.092 - INFO - Pass âœ“...  Passing result
